### PR TITLE
Minimal test to check that messages get black after sending them

### PR DESCRIPTION
**Pull request by jschirrmacher**
This uses chimp to send messages in the channel #general and waits until it gets black until the next is sent.

Invoke with:
`CHIMP_PATH=tests/performance npm run chimp-path`

### CODE REVIEW

File: tests/pageobjects/main-content.page.js [hunk 1]
```diff
@@ -78,6 +78,25 @@ class MainContent extends Page {
...
 	}
 
+	sendMessageOptimistic(text) {
+		this.setTextToInput(text);
+		this.sendBtn.click();
+	}
+
+	waitForMessageArrival(text, timeout) {
+		browser.waitUntil(function() {
+			browser.waitForVisible('.message:last-child .body', timeout);
```
Comment (mrsimpson) [inline tests/pageobjects/main-content.page.js]: What's the selector for the gray messages? This one looks pretty common for both...

File: tests/chimp-config.js [hunk 2]
```diff
@@ -113,7 +113,7 @@ module.exports = {
 		reporter: 'spec',
 		slow: 100,
 		// retries: 3,
-		bail: true, // bail after first test failure
+		bail: false, // bail after first test failure
```
[the remark this answers is not in the document]
Comment (jschirrmacher) [reply tests/chimp-config.js]: You're right - I didn't adapt the comment, but will.
I understand `bail` in the sense of junping off - which is what it does when set to true.

File: tests/performance/messages.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+// test data imports
+import { username, email, password } from '../data/user.js';
+import { checkIfUserIsValid } from '../data/checks';
+
+function singleMessage(index) {
+	it('Message sending should work in less than 2 seconds', () => {
+		const text = `how long is the round-trip for message #${ index }?`;
+		mainContent.sendMessageOptimistic(text);
+		mainContent.waitForMessageArrival(text, 1000);
```
[the remark this answers is not in the document]
Comment (jschirrmacher) [reply tests/performance/messages.js]: It seems I worked too long yesterday... you're right... again.

File: tests/pageobjects/main-content.page.js (hunk 1 above)
Comment (jschirrmacher) [reply tests/pageobjects/main-content.page.js]: Good question... this PR is in fact still a draft... I need to find a way to provoke the grey message to get the correct selector.

**Comment by mrsimpson** [comment]
Closing this PR as this is not current anymore

**Pull request closed without merging**